src/java.base/share/classes/java/net/InterfaceAddress.java

```diff
@@ -62,6 +62,9 @@ public InetAddress getAddress() {
      * <p>
      * Only IPv4 networks have broadcast address therefore, in the case
      * of an IPv6 network, {@code null} will be returned.
+     * <p>
+     * Certain IPv4 addresses, such as the loopback address, do not support
+     * broadcasting and will also result in {@code null} being returned.
```

Comment:
```suggestion
     * Certain network interfaces, such as the loopback interface, do not support
     * broadcasting and will also return {@code null}.
```

Maybe the class level API documentation should also be updated something like:

```
 * This class represents a Network Interface address. In short it's an
 * IP address, a subnet mask as well as a broadcast address when the address is
 * IPv4 and the interface supports broadcasting.
 * An IP address and a network prefix length in the case
 * of IPv6 address.
```

It would be good to have @Michael-Mc-Mahon approve the wording.

Reply:
Thanks Daniel, I have commited the above change in wording to the get broadcast method and updated the CSR to reflect (JDK-8356002). I will await confirmation on the class level API documentation before updating.

Reply:
I think the original wording of the class level doc was phrased awkwardly. I'd like to suggest the following:
```
 * This class represents a Network Interface address. In the case of
 * IPv4, this comprises the IP address, a subnet mask and a broadcast 
 * address if the interface supports broadcast. In the case of IPv6, 
 * it comprises the IP address and a network prefix length.

```

Reply:
Thanks Michael! I agree this would be much better.

Reply:
Thanks both, the class level API documentation has been updated and CSR modified to reflect.